### DISCUSSION TITLE
Get-PnPAppInfo : The term 'Get-PnPAppInfo' is not recognized as the name of a cmdlet

### DISCUSSION THREAD

**Discussion by ghost** [pnp/powershell#1120] [Q&A] [answered]
I'm planning to install an app on my SharePoint 365 site via PowerShell using this example ([the GUID below is from the example in the documentation](https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/install-pnpapp?view=sharepoint-ps#example-2)):
`Install-PnPApp -Identity 99a00f6e-fb81-4dc7-8eac-e09c6f9132fe -Scope Site`

However in order to get the identity of the app I am trying to use:
`Get-PnPAppInfo -Name "App Display Name"`

But this returns the following:
> Get-PnPAppInfo : The term 'Get-PnPAppInfo' is not recognized as the name of a cmdlet, function, script file, or
> operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
> again.
> At line:1 char:1
> + Get-PnPAppInfo -Name "Nintex Forms for Office 365"
> + ~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (Get-PnPAppInfo:String) [], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException

I have installed the PnP.PowerShell prior to using Get-PnPAppInfo:
`Install-Module -Name PnP.PowerShell`

Failing that how else can I get the ID of a SharePoint 365 app?

**Comment by gautamdsheth** [comment, marked as answer]
You need to use `Get-PnPApp` cmdlet to get the app info. You are using `Get-PnPAppInfo` which doesn't exist, hence the error.

**Reply by ghost** [reply]
Perfect thanks!

Incidentally, Get-PnPAppInfo is still part of the [Microsoft documentation](https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/get-pnpappinfo?view=sharepoint-ps).

**Reply by veronicageek** [reply]
@gautamdsheth - The `Get-PnPAppInfo` does exist in the latest update + docs. Should it be removed?
